src/Tests/Framework/MockData/Project.cs

```diff
@@ -22,6 +22,7 @@ public class Project
 		public Dictionary<string, Metadata> Metadata { get; set; }
 		public SimpleGeoPoint Location { get; set; }
 		public int? NumberOfCommits { get; set; }
+		public int? NumberOfContributors { get; set; }
```

Comment:
This addition caused inferred mapping in `PutMappingApiTests` to fail. Will update PR with fix